src/main/java/uk/ac/cam/cl/dtg/isaac/dto/TestCaseDTO.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2019 Meurig Thomas
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ * 		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.cam.cl.dtg.isaac.dto;
+
+import uk.ac.cam.cl.dtg.segue.dos.content.Choice;
+import uk.ac.cam.cl.dtg.segue.dos.content.Content;
+
+/**
+ * DTO for holding example user submitted questionChoices and their expected and actual values when run against our
+ * validators
+ */
+public class TestCaseDTO {
+    Choice choice;
+    Boolean expected;
+    Boolean actual;
+    Content explanation;
```

Comment:
This class seems similar (from memory - although I could be wrong) to the questionResponse type of objects. Is there something nice we can do to reuse / extend them?